### DISCUSSION TITLE
Map orientation and moving map

### DISCUSSION THREAD

**Discussion by benniAvia** [Akaflieg-Freiburg/enroute#434] [Q&A] [answered]
Hi all, 

Perhaps, I am doing something wrong, can somebody help me? 

I have started to use Enroute on the iPhone 11 Pro Max and iPad (10" 2022 model). 

However, I have some challenges:

1. On the iPad I do not see the map moving at all, the icon is still stuck at the airport where I took off from.  I restarted the app, then it moves a bit. 

2. Then, the map orientation: I have tried to have it track up, but it does not show the north correctly, it is showing a bit weird directions. 

3. Finally, the arrow does not come on... it came just once.

Any idea? Is the aircraft I fly too slow? A DV20? Do I need something faster? 😁

**Comment by schneisi** [comment, marked as answer]
No, a DV20 is not too slow ;)

does your iPad have cellular or is it WiFi only?
This is important, because the WiFi only models have no built-in GPS

**Comment by benniAvia** [comment]
Hi Simon

Ah, I see! It is a 7th generation iPad bought 2022 (I think)!

But as I check, it does not see to have a Cellular. 

But it does say Location, and if I use any location based app, it seems to pick up the right location though... ?

Weirdly though, it shows the small triangle at the top when the GPS is being used (as in the iPhone (same icon).
So, now wondering!

tagging @schneisi

**Reply by schneisi** [reply]
As far as I know, iPads without GPS determine their location mainly through WiFi-based positioning.
These iPads detect nearby WiFi networks and use the signal strengths to estimate their location. This works because the locations of many WiFi networks are known, allowing the iPad to compare the signals it detects against a database to figure out where it is. This method is fairly accurate in areas with many WiFi networks.

However, in places without WiFi signals, like high above the ground, the iPad simply fails to determine its location.

Please take the details with a grain of salt, I am not an expert in this area. But this is more or less how it works.

**Comment by benniAvia** [comment]
Thanks Simon! I get it now, appreciate the help. 

Time to get a new device anyhow! 😊
